site/src/pages/UserSettingsPage/NotificationsPage/NotificationsPage.tsx

```diff
@@ -60,6 +62,27 @@ export const NotificationsPage: FC = () => {
 	const updatePreferences = useMutation(
 		updateUserNotificationPreferences(user.id, queryClient),
 	);
+	const [searchParams] = useSearchParams();
```

Comment:
@BrunoQuaresma could you please review this?

Reply:
Sure thing!